cdap-docs/reference-manual/source/http-restful-api/logging.rst

```diff
@@ -13,7 +13,7 @@ Logging HTTP RESTful API
 
 Downloading Logs
 ----------------
-Logs emitted by any of the *Flows*, *Procedures*, *MapReduce* jobs, or
+Logs emitted by any of the *Flows*, *Procedures*, *MapReduce* programs, or
```

Comment:
don't we need Spark in this list, too? And because Procedures are deprecated, remove that?